db/yaml/maskrcnn/20201221195937.yaml

```diff
@@ -11,13 +11,13 @@ arguments:
       value: annotation-dump/sample_dataset
```

Comment:
Bit confused about this file, are you deleting an old migration file here?

Reply:
@rushtehrani This migration adds the `select.nodepool` parameter